src/index.js

```diff
@@ -1,17 +1,37 @@
 import fs from 'fs-extra';
 import path from 'path';
+import filedirname from 'filedirname';
+const [__filename, __dirname] = filedirname(new Error());
```

Comment:
```suggestion
const [__dirname] = filedirname(new Error());
```

Do we need `__filename` here?